### ISSUE TITLE
Fullscreen flickering on Transition

### ISSUE THREAD

**Issue by PabloNeirotti** [Ancurio/mkxp#128]
Hi there!

Since I first started playing around with MKXP, I noticed that in-game Transitions (using such screen provided by Screen, is it?) make the screen flicker to black, artifacts, or probably previos graphics displayed by the graphic card before.

This issue occurs only during transitions between Scenes, if you may, and exclusively on Fullscreen. This is tested on Windows.

Any ideas why this may be happening? nVidia drivers for GeForce 660M are up to date.

Thank you!

**Comment by Ancurio** [comment]
Hmm, could you make a screenshot or video of that?

**Comment by PabloNeirotti** [comment]
Yes! Finally managed to upload it.

https://youtu.be/IvCyw9_pPmY

On that video I try both Fullscreen and then Windowed so you can see the difference.

Thank you! :D

**Comment by Ancurio** [comment]
Cool, thanks. That looks like the transition isn't happening at all. It's weird that this is happening on Nvida since their drivers are known to be the best..

But it's better to test with a clean testcase. Here is one I wrote, which tests both normal and bitmap transitions: https://www.dropbox.com/s/ji61ces5a9ssxcm/buggy_trans_nvidia.zip?dl=0  
Point `gameFolder` at "buggy_trans_nvidia" and set `customScript=script.rb`.

**Comment by PabloNeirotti** [comment]
Thanks for the response!

I just tried this, and worked flawlessly. I also added blur in the mix thinking that could be causing it (since RPG Maker blurs the map view before switching), but works fine both fullscreen and windowed mode.

Although I just removed blur altogether from the in game transition and it still flickers. Wild guess, but it isn't it. I just realized this also occurs on the main menu of the game, and there is no map view or blurring there...

Hmmm this makes me ponder...
